.travis.yml

```diff
@@ -1,11 +1,27 @@
 language: cpp
-
+sudo: required
+dist: trusty
+os:
+  - linux
+  - osx
 compiler:
   - clang
   - gcc
+rvm:
+  - 1.9.3
+  - 2.0
+  - 2.2
```

Comment:
Yes - I noticed that it doesn't work. And agree that it is probably because it's not ruby project for travis. I left it there to think about it later, but then forgot about that...
I agree that we shouldn't be concerned with testing multiple Ruby versions. 
I believe we can just depend on ruby version preinstalled on CI (if it works)